src/bindings/c/include/openvino/ov_node.h

```diff
@@ -0,0 +1,112 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief This is a header file for ov_model C API, which is a C wrapper for ov::Node class.
+ *
+ * @file ov_model.h
+ */
+
+#pragma once
+
+#include "ov_common.h"
+#include "ov_partial_shape.h"
+#include "ov_shape.h"
+
+typedef struct ov_output_const_node ov_output_const_node_t;
+typedef struct ov_output_node ov_output_node_t;
+
+/**
+ * @struct ov_output_node_list_t
+ * @brief Reprents an array of ov_output_nodes.
+ */
+typedef struct {
+    ov_output_const_node_t* output_nodes;
+    size_t num;
+} ov_output_node_list_t;
+
+// Node
+/**
+ * @defgroup node node
+ * @ingroup openvino_c
+ * Set of functions representing of Model and Node.
+ * @{
+ */
+
+/**
+ * @brief Get the tensor name of ov_output_node list by index.
+ * @ingroup node
+ * @param nodes A pointer to the ov_output_node_list_t.
+ * @param idx Index of the input tensor
+ * @param tensor_name A pointer to the tensor name.
+ * @return Status code of the operation: OK(0) for success.
+ */
+OPENVINO_C_API(ov_status_e) ov_node_get_any_name_by_index(ov_output_node_list_t* nodes, size_t idx, char** tensor_name);
```

Comment:
I guess it should be get_name_by_index

Reply:
C++ method uses get_any_name(), so C adopts the same name.

Reply:
We don't have in C++ api get_any_name with index.
Can we change this method to
```suggestion
OPENVINO_C_API(ov_status_e) ov_node_get_any_name(ov_output_node_t* node char** tensor_name);
```

Reply:
ok

Reply:
After re-think, i think maybe we still need keep the second api to make user can access each node in the node vector/list that return by of ov::model::inputs() or ov::Model::outputs().
Due to we encapsulated std::vector<ov::Output<const ov::Node>>into a C list, each item is an opaque handle, we cannot access it one by one in C space, only can access in C++ space, so need a interface to pass index.

Reply:
ok, we can delete the api that can get tensor name for node in ov_output_node_list_t, but corresponding, we need add a new api to get node from  ov_output_node_list_t.
Do you think it is reasonable?

Reply:
In C structure, we use a list to store all nodes returned by ov::model::inputs() and ov::model::outputs(),  it is reasonable to get each node name by list index, why need to query it port and then get name by port?